.travis.yml

```diff
@@ -1,97 +0,0 @@
-dist: xenial
-
-services:
-  - postgresql
-
-env:
-  global:
-    - ENVIRONMENT=testing
-    - LIBRETIME_LOG_DIR=/tmp/log/libretime
-    
-before_install:
-  - sudo add-apt-repository -y ppa:libretime/libretime
-  - sudo apt-get -q update
-
-addons:
-  apt:
-    packages:
-      - gstreamer1.0-plugins-base
-      - gstreamer1.0-plugins-good
-      - gstreamer1.0-plugins-bad
-      - gstreamer1.0-plugins-ugly
-      - libgirepository1.0-dev
-      - liquidsoap
-      - liquidsoap-plugin-faad
-      - liquidsoap-plugin-lame
-      - liquidsoap-plugin-mad
-      - liquidsoap-plugin-vorbis
-      - python3-gst-1.0
-      - silan
-      - gcc
-      - gir1.2-gtk-3.0
-      - python3-gi
-      - python3-gi-cairo
-      - python-cairo
-      - pkg-config
-      - libcairo2-dev
-
-jobs:
-  include:
-#    - language: php
-#      php: 7.3
-#      stage: test
-#    - language: php
-#      php: 7.2
-#      stage: test
-    - language: php
-      php: 7.0
-      stage: test
-    - language: php
-      php: 5.6
-      stage: test
-#    - language: python
-#      python: 3.8
-#      stage: test
-#    - language: python
-#      python: 3.6
-#      stage: test
-    - language: python
-      python: 3.5
-      stage: test
-    - stage: deploy
-      addons:
-        apt:
-          packages:
-            dos2unix
-      script:
-        ./travis/release.sh
-      deploy:
-        - provider: releases
-          token: $GITHUB_TOKEN
-          file: build/libretime-*.tar.gz
-          edge: true
-          on:
-            tags: true
```

Comment:
I think we'll also want to add a workflow that covers these release bits which where responsible for uploading a fat tarball (ie. with packages php deps) to the GitHub releases page.

Reply:
Of course. How often should it run?

Reply:
On tags on master

Reply:
On *any* tag, or just tags that include "v." or versioning information?

Reply:
Any tags or a regex for `\d+\.\d+\.\d+(~\s+\d+)*` (I think, that is untested)

Reply:
> \d+\.\d+\.\d+(~\s+\d+)*

I don't understand that at all. Sorry. 😢 

Do tags *only* get applied to releases?

Reply:
Actually it should be `\d+.\d+.\d+(~\s+\.\d+)*`. It should match things like 3.0.0~alpha.5, but yeah, we only creat tags for releases. It also matches things like 3.0.0

Reply:
We're not currently using a v prefix so just running it on every tag would be fine IMO.

If we change anything related to tagging we can refactor the workflow to use a pattern then.